Analysis/python/CreateSpectralHists.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
I don't think that we want to run multiple instances of `CreateSpectralHists` at the same machine in parallel. This would also require some mechanism to track jobs which are failed (for one reason or another). I propose to keep things simple for the moment: run CreateSpectralHists consecutively one folder at the time and use implicit multithreading for reading ROOT file (see comment above). If it doesn't work sufficiently fast, we can think about more elaborate parallelization ways.